Vic2ToHoI4/Source/Mappers/CasusBelli/CasusBellis.cpp

```diff
@@ -0,0 +1,13 @@
+#include "CasusBellis.h"
+
+
+
+std::string Mappers::CasusBellis::getWarGoalFromCasusBelli(const std::string& casusBelli) const
+{
+	if (const auto& mapping = casusBelliToWarGoalMap_.find(casusBelli); mapping != casusBelliToWarGoalMap_.end())
+	{
+		return mapping->second;
+	}
+
+	return "topple_government";
```

Comment:
I am unsold, but it may depend on usage. I guess you trained me to always return nullopt and handle it on the receiving end.

Reply:
Normally I would, but in this case the default is an intended behavior rather than a fallback for an error.